### DISCUSSION TITLE
Some files have been left conflicted! Please resolve them by "Pick a file to resolve conflict".

### DISCUSSION THREAD

**Discussion by wangweitung** [vrtmrz/obsidian-livesync#239] [Q&A]
Hi everyone, I often got this notice while I am using live sync, even if I rebuild and fetch the database, it still occurs.

~~~
2023/7/8 11:20:53->Some files have been left conflicted! Please resolve them by "Pick a file to resolve conflict". The list is written in the log.
2023/7/8 11:20:53->Conflicted: ix:TS/PLUGIN_DATA/obsidian-git.md
~~~

Could you please tell me how to solve this?

Or give  me an  instruction to solve it quickly thank you guys!

**Comment by vrtmrz** [comment]
Hello! Thank you for starting the discussion!

The conflicted file is the customization sync data. And, `TS` following `ix:` indicates that the `Device name`.
I am asking for trouble, but could you please make sure that all of the `Device name` are unique between your synchronised devices?

And, possibly using `Pick a file to resolve conflict`(this can be shown from the command palette) could solve the conflict.

**Reply by wangweitung** [reply]
yes， I have 5 devices，each one of them has an unique name.
and thanks for the answer，I will try  from the command line。

I have another question： it seems difficult for live sync to sync big file ，eg: I put a font file in the vault , but live sync just sync the name , instead of the file ，and the db file in couhcdb becomes much  more larger（when without the font file  7M, after adding the font file , 70M）.

so  live sync is only for sync .md file , am I right？
